server/controllers/TasksController.js

```diff
@@ -8,7 +8,7 @@ import { tasksService } from '../services/TasksService'
 //PUBLIC
 export class BoardsController extends BaseController {
   constructor() {
-    super("api/boards")
```

Comment:
check to make sure we need api/boards in the tasksController, I am not sure it should be there! Thanks!